### ISSUE TITLE
Unused Variable for import used in object literal property value shorthand

### ISSUE THREAD

**Issue by unional** [Microsoft/vscode-tslint#30]
`typescript@1.8.2`
`tslint@3.5.0`

```ts
import funcA from './funcA'; // Unused Variable 'funcA'

// either
export default {
  funcA
}

// or
let a = {
  funcA
};
```

It is flagged by tslint through language service as a warning:
![image](https://cloud.githubusercontent.com/assets/3254987/13404129/248b84ec-decd-11e5-972e-4fe36eb028c8.png)
![image](https://cloud.githubusercontent.com/assets/3254987/13404189/5bd65b66-decd-11e5-9726-b4cc2bd2642b.png)

The warning is gone when shorthand is not used:
![image](https://cloud.githubusercontent.com/assets/3254987/13404204/6f3d8fda-decd-11e5-8fd9-04866d76f85b.png)

reference:
https://github.com/palantir/tslint/issues/1005

**Comment by egamma** [comment]
I'm sorry, but I cannot reproduce given the above steps.
